pkg/types/clusterconfig/clusterconfig.go

```diff
@@ -141,6 +142,19 @@ var UserValidation = &cr.StructValidation{
 				return VolumeTypeFromString(str), nil
 			},
 		},
+		{
+			StructField: "Tags",
+			StringMapValidation: &cr.StringMapValidation{
+				AllowExplicitNull: true,
+				AllowEmpty:        true,
+				Validator: func(val map[string]string) (map[string]string, error) {
```

Comment:
Consider adding a field to the validator like `ConvertNilToEmpty: bool`, but not a big deal if you think that's more trouble than it's worth